django/user/views.py

```diff
@@ -1,13 +1,22 @@
-from django.shortcuts import render
-from django.http import HttpResponse, JsonResponse
+from django.views.generic import DetailView, ListView, CreateView
+from django.urls import reverse_lazy
 from .models import User
+from .forms import UserForm
 
 
-def say_hello(request):
-    return HttpResponse('Hello, users!')
+class UserListViews(ListView):
+    model = User
+    template_name = 'user_list.html'
+    context_object_name = 'users'
 
 
-def get_user_list(request):
-    users = list(User.objects.values())
-    return JsonResponse(users, safe=False)
+class UserDetailView(DetailView):
+    model = User
+    template_name = 'user_detail.html'
+    context_object_name = 'user'
 
+
+class UserFormView(CreateView):
```

Comment:
nit: recording to previous namings
```suggestion
class UserCreateView(CreateView):
```